README.md

```diff
@@ -51,55 +93,81 @@ public class Address {
   @Example("2023-12-03T10:15:30+01:00[Europe/Madrid]")
   private ZonedDateTime deliveryTime;
 
+  @Example("Sep 27, 2022, 11:02:00 AM")
+  private Date creationDate;
+
   @Example("Jose")
   private String name;
 
   @Max(12)
   @Min(1)
-  @Example("4")
   private int number;
 
-  private ZonedDateTime init;
-
+  @Example("4")
+  private long aLong;
 
+  private ZonedDateTime init;
+  
   private City city;
 
 }
 ```
-Other annotations you can use in order to configure value and builder generation are `@Max` `@Min`. Which will indicate the minimum and maximum values that property can have. In case of collections it will delimitate the number of elements that collection will contain.
 
-### How to use the builders
+### Builder
+
+Once the code is compiled, builder will be generated and available under `generated-sources` in your build directory.
+
+You will need to add the required import, and make use of the related builder class.
+
+#### Example
 
-Once the code is compiled, builders will be generated and available to build Pact Json Bodies. Just only need to add the import you required and add the values you want to overwrite
-So this code 
 ```java
-    personBuilder
-        .setAddress(new com.sngular.model.AddressBuilder()
-             .setCity(new com.sngular.model.CityBuilder()
-                      .setCoords(BigDecimal.valueOf(8.7))
-                      .setCityName("Ourense")))
-    .build();
-```
-will generate this output
-
-```json
-{
-  "address":
-  {
-    "city":
-    {
-      "cityName":"Ourense",
-      "coords":8.7,
-      "isCapital":true,
-      "objectList":["2023-10-25[[+02:00]]"]
-    },
-    "deliveryTime":"2023-12-03T10:15:30.000000.000+01:00[Europe/Madrid]",
-    "init":"2023-10-25T17:47:09.637165.637+02:00[Europe/Madrid]",
-    "name":"Jose",
-    "number":4
-  },
-  "age":1572804885,
-  "name":"7z2",
-  "wallet":1.6901720963006214E+308
+import com.sngular.model.AddressBuilder;
+        
+AddressBuilder addressBuilder = new AddressBuilder();
+DslPart bodyDslPart = addressBuilder.build();
+
+
+@Pact(consumer = "consumer-poc", provider = "provider-poc")
+public RequestResponsePact getStudents(PactDslWithProvider builder) {
+        AddressBuilder addressBuilder = new AddressBuilder();
+        DslPart bodyDslPart = studentBuilder.build();
+        
+        return builder.given("Address exist")
+        .uponReceiving("get all address")
+        .path("/address/")
+        .method("GET")
+        .willRespondWith()
+        .status(200)
+        .headers(Map.of("Content-Type", "application/json"))
+        .body(bodyDslPart)
+        .toPact();
 }
-```
\ No newline at end of file
+```
+
+## Roadmap
+
+Roadmap available under [GitHub Projects section](https://github.com/orgs/sngular/projects/2).
```

Comment:
This link does not seem to exist.

Reply:
It does exist although it is still private. Pending to make it public.